.gitattributes

```diff
@@ -1,4 +1,3 @@
 sql/parser/sql.go -diff
 ui/embedded.go -diff
-*.pb.go -diff
-*.pb.gw.go -diff
+*.pb.* -diff
```

Comment:
no, `-diff` doesn't, it just disables patch and diff generation.
`merge` is what controls merge behavior, so `-merge` (or `merge=binary`) will cause merges to leave the working copy but mark it conflicted.

The `binary` macro is `-diff -merge -text`.